src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+const booksSlice = createSlice({
+  name: 'addBooks',
+  initialState,
+  reducers: {
+    addBooks(state, action) {
+      state.push(action.payload);
+    },
```

Comment:
- [ ] Great job implementing required changes from the previous reviewer. However, there's a little stuff to look into here. There's `books` array of object inside your state which is equivalent to initialState. Pushing  the book object into state directly will not fill the book array of objects. Baiscally it should be `state.books.push(action.payload)`